groclient/experimental.py

```diff
@@ -3,31 +3,65 @@
 
 
 class Experimental(GroClient):
-    """
-    Experimental class to consume v2prime data.
-    """
+    """Experimental class to consume v2prime data."""
 
     def get_data_points(self, **selections):
-        """
-        Get all the data points for a given selection from /v2prime/data.
+        """Get all the data points for a given selection from /v2prime/data.
+
+        This function is a part of an experimental class and subject to change.
+
+        Parameters
+        ----------
+        metric_id : integer
+            How something is measured. e.g. "Export Value" or "Area Harvested"
+        item_ids : integer or list of integers
+            What is being measured. e.g. "Corn" or "Rainfall"
+        region_ids : integer or list of integers
+            Where something is being measured e.g. "United States Corn Belt" or "China"
+        partner_region_ids : integer or list of integers, optional
+            partner_region refers to an interaction between two regions, like trade or
+            transportation. For example, for an Export metric, the "region" would be the exporter
+            and the "partner_region" would be the importer. For most series, this can be excluded
+            or set to 0 ("World") by default.
+        source_id : integer
+        frequency_id : integer
+        unit_id : integer, optional
+        start_date : string, optional
+            All points with end dates equal to or after this date
+        end_date : string, optional
+            All points with start dates equal to or before this date
+        coverage_threshold: float, optional
+            Custom threshold on the coverage of geospatial data. Value should be between 0 and 1.
+        stream: bool, optional
```

Comment:
let's remove stream for now - it's not really making any difference to end users , we can add it back when we support real streaming between API & client

Reply:
Thanks. Done.